TIG/TIG/Feature/Home/More/WeeklyRepeatView.swift

```diff
@@ -71,13 +72,24 @@ struct WeeklyRepeatView: View {
                     })
                 }
             }
-        })
+            
+            ToolbarItem(placement: .topBarLeading) {
+                Button(action: {
+                    homeViewModel.effect(.exitRepeatView)
+                    self.presentationMode.wrappedValue.dismiss()
```

Comment:
```suggestion
                    dismiss()
```
이렇게 간단하게 호출할 수 있습니다.

Reply:
수정하였습니다!